admiral/pkg/clusters/serviceentry.go

```diff
@@ -290,6 +292,16 @@ func modifyServiceEntryForNewServiceOrPod(
 				Namespace: namespace,
 				Type:      common.Deployment,
 			}
+			if common.IsVSBasedRoutingEnabled() {
+				err := generateIngressVirtualServiceForDeployment(deployment, sourceIngressVirtualService)
+				if err != nil {
+					err = fmt.Errorf(ingressVSGenerationErrorMessage + " w%" + err.Error())
+					ctxLogger.Errorf(common.CtxLogFormat, "generateIngressVirtualServiceForDeployment",
+						deployment.Name, deployment.Namespace, clusterId, err.Error())
+					modifySEerr = common.AppendError(modifySEerr, err)
```

Comment:
Can we make sure the cluster name is present in the error message

Reply:
@vinay-g , `clusterId` is the cluster name. The variable name could have been more intuitive :)

Reply:
oh my bad, you meant the error message. Sorry yeah, I'll make the change